src/dotgraph.cpp

```diff
@@ -301,7 +301,7 @@ void DotGraph::writeGraphHeader(TextStream &t,const QCString &title)
          "labelfontname=\"" << fontName << "\","
          "labelfontsize=\"" << fontSize << "\"];\n";
   t << "  node [fontname=\"" << fontName << "\","
-         "fontsize=\"" << fontSize << "\",shape=record];\n";
+         "fontsize=\"" << fontSize << "\",shape=\"box\"];\n";
```

Comment:
Comas and quotes are optional for graphviz and are confusing in embedded code. IMHO better just `shape=box`

Reply:
BTW I have fix to make attributes configurable.

Reply:
I think it would be better to have the `"` present as it is more conform with the other code like `fontsize`, `fontname` etc., though the word `box` is just one word and `fontsize` and `fontame` could have e.g a space in it which  makes here the quotes mandatory.

I don't get the point of "are confusing in embedded code", cannot think of any confusion.

Reply:
Regarding https://github.com/doxygen/doxygen/pull/9390#discussion_r894450582 (configurable), depends to which level it is configurable but I'm not sure about it. Maybe you should make as proposed pull request out of what you have.

Reply:
I am proposing #7839 for already two years and one day. Now I spit it into small chunks. The next chunk is #9353.